app/src/main/java/rick_and_morty/data/model/RickAndMortyResponseDto.kt

```diff
@@ -0,0 +1,13 @@
+package rick_and_morty.data.model
+
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class RickAndMortyResponseDto(
+    @Json(name = "info")
+    val infoDto: InfoDto? = null,
+    @Json(name = "results")
+    var results: List<CharacterResultsDto>? = null
```

Comment:
same here about default values. Also any reason this is `var` and not `val`?

Reply:
done